practitioner-service/src/main/resources/bootstrap-local.yml

```diff
@@ -5,4 +5,8 @@ spring:
       discovery:
         enabled: false
     discovery:
-      enabled: false
\ No newline at end of file
+      enabled: false
+
+ribbon:
```

Comment:
Lets add a comment here to explain what it does.

Reply:
Done